packages/client/libclient-py/quickmpc/pandas/share_data_frame.py

```diff
@@ -151,27 +178,89 @@ def join_list(self, others: List["ShareDataFrame"], *, debug_mode=False)\
                               True, ShareDataFrameStatus.EXECUTE)
 
     def sum(self, columns: list) -> "ShareDataFrame":
+        """列の総和を取得する
+
+        Parameters
+        ----------
+        columns: list
+            計算に用いる列番号(1-index)
+
+        Returns
+        ----------
+        Result
+            結果のDataFrame
+        """
         res = self.__qmpc_request.sum([self.__id], columns)
         return ShareDataFrame(res.job_uuid, self.__qmpc_request,
                               True, ShareDataFrameStatus.EXECUTE)
 
     def mean(self, columns: list) -> "ShareDataFrame":
+        """列の平均を取得する
+
+        Parameters
+        ----------
+        columns: list
+            計算に用いる列番号(1-index)
+
+        Returns
+        ----------
+        Result
+            結果のDataFrame
+        """
         res = self.__qmpc_request.mean([self.__id], columns)
         return ShareDataFrame(res.job_uuid, self.__qmpc_request,
                               True, ShareDataFrameStatus.EXECUTE)
 
     def variance(self, columns: list) -> "ShareDataFrame":
+        """列の分散を取得する
+
+        Parameters
+        ----------
+        columns: list
+            計算に用いる列番号(1-index)
+
+        Returns
+        ----------
+        Result
+            結果のDataFrame
+        """
         res = self.__qmpc_request.variance([self.__id], columns)
         return ShareDataFrame(res.job_uuid, self.__qmpc_request,
                               True, ShareDataFrameStatus.EXECUTE)
 
     def correl(self, columns1: list, columns2: list) -> "ShareDataFrame":
+        """列同士の分散を取得する
+
+        Parameters
+        ----------
+        columns1: list
```

Comment:
I prefer to be as detailed as `list[int]` or `list of int` instead of `list`.
I particularly prefer `list[int]`, but the style guide may recommend `list of int`.

Reply:
I added it! [a150c5e](https://github.com/acompany-develop/QuickMPC/pull/270/commits/a150c5e9d4b61599522b4afb1d70f2141ebda310)
`list[int]` is not available in python 3.7, so I chose `List[int]`.